arsceneview/src/main/java/io/github/sceneview/ar/node/ArModelNode.kt

```diff
@@ -100,6 +100,8 @@ open class ArModelNode : ArNode {
             }
         }
 
+    var updateUnanchoredPose = true
```

Comment:
Can you add documentation here?

Reply:
Please wait a little bit before merging

Reply:
Question: Shouldn't this case be not relevant since the node position is only possible on an anchored node or not when its an ArModelNode?

Reply:
When moving the node and detaching the anchor, the node is positioned back at the center of the screen. However, there is no such problem with an `ArNode`.

Reply:
It's line 146 in ArModelNode: onArFrameHitResult(hitTest(arFrame)). This hitTest uses the middle of the screen and overrides the pose set by the ArNodeManipulator.
@ThomasGorisse You can merge the PR when it works for you.

Reply:
@ThomasGorisse Can I merge it now?

Reply:
Yes, I rebased it and fixed the merge conflict

Reply:
I finish pushing. Can you rebase and merge?
Sorry for the time it took but those commits quite improve SceneView